Python 3: DEFAULT_SEQUENCER.sequence "clean" bytes instead of string

Sub-task of #25 
```
bgp/bgp_test.py:17: in <module>
    genome = DEFAULT_SEQUENCER.sequence(book)
bgp/__init__.py:105: in sequence
    sq.pipeline[p].run(sq.book)
bgp/modules/terms.py:78: in run
    self.terms = self.fulltext_to_ngrams(
bgp/modules/terms.py:104: in fulltext_to_ngrams
    tokens = [t.strip() for t in clean(fulltext).split(b' ') if t and t not in stop_words]
bgp/modules/terms.py:98: in clean
    return ''.join(c.encode("ascii", "ignore") for c in (
E   TypeError: sequence item 0: expected str instance, bytes found
```

--- a/bgp/bgp_test.py
+++ b/bgp/bgp_test.py
@@ -21,7 +21,7 @@ class TestSequencer(unittest.TestCase):
 
     def test_smoketest(self):
         book = ia.get_item(item_id)
-        genome = DEFAULT_SEQUENCER.sequence('hpmor')
+        genome = DEFAULT_SEQUENCER.sequence('arcadeflyer_the-ninja-kids')
     
     def test_copyright_page_detection(self):
         assert genome.results['pagetypes']['copyright_page'][0] == '0004'

--- a/bgp/bgp_test.py
+++ b/bgp/bgp_test.py
@@ -20,7 +20,6 @@ genome = DEFAULT_SEQUENCER.sequence(book)
 class TestSequencer(unittest.TestCase):
 
     def test_smoketest(self):
-        book = ia.get_item(item_id)
         genome = DEFAULT_SEQUENCER.sequence('arcadeflyer_the-ninja-kids')
     
     def test_copyright_page_detection(self):

--- a/bgp/modules/terms.py
+++ b/bgp/modules/terms.py
@@ -93,13 +93,13 @@ class NGramProcessor():
                            punctuation='!"#$%&\'()*+,.-;<=>?@[\\]^`{|}*'):
         stop_words = stop_words or {}
         def clean(fulltext):
-            return ''.join(c.encode("ascii", "ignore") for c in (
+            return ''.join(c.encode("ascii", "ignore").decode() for c in (
                 fulltext.lower()
                 .replace('. ', ' ')
                 .replace('\n-', '')
                 .replace('\n', ' ')
             ) if c not in punctuation)
-        tokens = [t.strip() for t in clean(fulltext).split(b' ') if t and t not in stop_words]
+        tokens = [t.strip() for t in clean(fulltext).split(' ') if t and t not in stop_words]
         return cls.tokens_to_ngrams(tokens, n=n) if n > 1 else tokens
 
 

